### PR TITLE
First emails redesign

### LINKED ISSUE #8214: Review invite user to an org email
Text and design are very weak
em

### LINKED ISSUE #8939: Review share dataset email
screen shot 2016-07-18 at 12 18 58

Part of #8889.
cc @urbanphes

### LINKED ISSUE #8940: Review share map email
screenshot_20160715_114950

Part of #8889.
cc @urbanphes

### LINKED ISSUE #8944: Review liked map email
screen shot 2016-07-18 at 12 28 16

Part of #8889.
cc @urbanphes

### LINKED ISSUE #8945: Review liked dataset email
screen shot 2016-07-18 at 12 33 04

Part of #8889.
cc @urbanphes

### LINKED ISSUE #8946: Review trending map mail
screen shot 2016-07-18 at 12 34 50

Part of #8889.
cc @urbanphes

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #8214 [Invite user to organization]
Closes #8939 [Review share dataset email]
Closes #8940 [Review share map email]
Closes #8944 [Review liked map email]
Closes #8945 [Review liked dataset email]
Closes #8946 [Review trending map mail]

- Invitation to join organization:
screen shot 2016-08-02 at 14 03 40

- Liked dataset:
screen shot 2016-08-02 at 12 16 47

- Liked map:
screen shot 2016-08-02 at 12 17 19

- Trending map:
screen shot 2016-08-02 at 12 30 37

- Shared dataset:
screen shot 2016-08-02 at 12 19 47

-Shared map:
screen shot 2016-08-02 at 12 33 38

CR @matallo @gfiorav

### CODE REVIEW

**Comment by xavijam** [comment]
Bravo! I'm looking forward to seeing them in production 💋

**Comment by matallo** [comment]
oeoeoe!

could we style the links? at least I can see two different colors

File: lib/static_maps_url_helper.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@ def url_for_static_map_without_request(username, request_protocol, visualization
         static_maps_image_url_fragment(visualization.id, map_width, map_height)
     end
 
+    def url_for_static_map_with_visualization(visualization, request_protocol, map_width, map_height)
+      static_maps_base_url(visualization.user.username, request_protocol) +
```
Comment (gfiorav) [inline lib/static_maps_url_helper.rb]: Keep an eye on Rollbar in case some nil user visualisation arrive here for some reason and create an exception.

**Comment by gfiorav** [comment]
👍 on my side for the backend part once 🍏

**Comment by MariaCheca** [comment]
@matallo you're right thanks! Plus, As we can't use custom fonts, we opted for using Helvetica, Arial and the rest of most common sans-serif in every system. What do you think about using Montserrat and Open Sans just in case the user has them installed?

**Comment by matallo** [comment]
not a fan of custom fonts for email, but that's not my call :)

**Comment by xavijam** [comment]
I like the idea of, if you have that font -> great, we render it. If not, the fallback one will be ready ;).

**Comment by MariaCheca** [comment]
@matallo I styled the links correctly and added the 'option' to render Montserrat and Open Sans in case it's possible, anything else?

**Comment by matallo** [comment]
👍

**Comment by MariaCheca** [comment]
✨

**Comment by juanignaciosl** [comment]
Can't duplication be avoided, as we said? U_U. Every mail has the same code again, having to do 6 times each change :(

**Comment by xavijam** [comment]
Yes please, @MariaCheca  let's make use of partials in the next PR (and if you know other better technique, let's use it).

**Comment by piensaenpixel** [comment]
@MariaCheca could you review this with @josecruz ?

**Comment by piensaenpixel** [comment]
late :(

**Comment by xavijam** [comment]
Hey, don't be sad, we can open any issue about details, improvements or problems. Let's congratulate that we have the first batch of new emails :).

**Comment by josecruz** [comment]
💚 💙 ❤️

**Pull request merged**